### PR TITLE
Mapped active shieldgens

### LINKED ISSUE #27747: Certain machines set to ACTIVE will not turn on in templates.
Notably shieldgens, though I'm sure other things that require ACTIVE to function (emitters, generators etc) likely will not work, either.

### PR DESCRIPTION
**Pull request by AnturK**
Fixes #27747

### CODE REVIEW

**Comment by WJohn** [comment]
Thank you!

**Comment by AnturK** [comment]
I'll add wall shield later to this.

File: code/game/machinery/shieldgen.dm [hunk 1]
```diff
@@ -86,23 +86,23 @@
 	req_access = list(GLOB.access_engine)
 	max_integrity = 100
 	obj_integrity = 100
-	var/active = 0
+	var/active = FALSE
 	var/list/deployed_shields = list()
```
Comment (Cyberboss) [inline code/game/machinery/shieldgen.dm]: Move the instantiation to Initialize

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/machinery/shieldgen.dm (hunk 1 above)
Comment (AnturK) [inline code/game/machinery/shieldgen.dm]: Sure.

**Pull request merged**